tools/Makefile

```diff
@@ -83,6 +83,9 @@ IRrecv.o : $(USER_DIR)/IRrecv.cpp $(USER_DIR)/IRrecv.h $(USER_DIR)/IRremoteESP82
 %_decode : $(COMMON_OBJ) %_decode.o
 	$(CXX) $(CPPFLAGS) $(CXXFLAGS) -lpthread $^ -o $@
 
+code_to_raw : $(COMMON_OBJ) code_to_raw.o
+	$(CXX) $(CPPFLAGS) $(CXXFLAGS) -lpthread $^ -o $@
+
```

Comment:
Maybe move down _decode and change to % to cover both cases

Reply:
I'm not sure I can easily do that with Make's wildcards. If you know how, I'm all ears.

Reply:
It wasn't that easy, but I think I got a working solution, pushed commit 8bb8c6e to PR branch

Reply:
Many thanks. Looks good to me.